boa3_test/tests/examples_tests/test_wrapped_neo.py

```diff
@@ -1,489 +1,596 @@
+        path, _ = self.get_deploy_file_paths('wrapped_neo.py')
+        runner = NeoTestRunner(runner_id=self.method_name())
 
-        self.run_smart_contract(engine, path, 'symbol')
-        wrapped_neo_address = engine.executed_script_hash.to_array()
+        invokes = []
+        expected_results = []
 
-        engine.reset_engine()
-        nef_path, _ = self.get_deploy_file_paths_without_compiling(path)
-        engine.add_contract(nef_path)
         allowed_amount = 10
+        owner_script_hash = self.OWNER.script_hash.to_array()
+        test_account_1 = self.OTHER_ACCOUNT_1
+        test_account_1_script_hash = test_account_1.script_hash.to_array()
+        test_account_2 = self.OTHER_ACCOUNT_2
+        test_account_2_script_hash = test_account_2.script_hash.to_array()
+        test_account_3 = self.OTHER_ACCOUNT_3
+        test_account_3_script_hash = test_account_3.script_hash.to_array()
+
+        runner.add_gas(self.OWNER.address, self.GAS_TO_DEPLOY)
+        runner.add_gas(test_account_3.address, self.GAS_TO_DEPLOY)
+        runner.deploy_contract(path, account=self.OWNER)
+        runner.update_contracts(export_checkpoint=True)
 
         # OWNER will give zNEO to OTHER_ACCOUNT_3 so that it can approve another contracts
-        result = self.run_smart_contract(engine, path, 'transfer',
-                                         self.OWNER_SCRIPT_HASH, self.OTHER_ACCOUNT_3, 10_000_000, None,
-                                         signer_accounts=[self.OWNER_SCRIPT_HASH])
-        self.assertEqual(True, result)
-        transfer_events = engine.get_events('Transfer')
-        self.assertEqual(2, len(transfer_events))
-        self.assertEqual(3, len(transfer_events[1].arguments))
-
-        sender, receiver, amount = transfer_events[1].arguments
-        if isinstance(sender, str):
-            sender = String(sender).to_bytes()
-        if isinstance(receiver, str):
-            receiver = String(receiver).to_bytes()
-        self.assertEqual(self.OWNER_SCRIPT_HASH, sender)
-        self.assertEqual(self.OTHER_ACCOUNT_3, receiver)
+        invokes.append(runner.call_contract(path, 'transfer', owner_script_hash, test_account_3_script_hash,
+                                            10_000_000, None))
+        expected_results.append(True)
+        runner.execute(account=self.OWNER)
+        self.assertEqual(VMState.HALT, runner.vm_state, msg=runner.error)
+
+        runner.run_contract(path, 'transfer', owner_script_hash, test_account_3_script_hash, 10_000_000,
+                            None, account=self.OWNER)
+
+        transfer_events = runner.get_events('Transfer')
+        self.assertEqual(1, len(transfer_events))
+        self.assertEqual(3, len(transfer_events[0].arguments))
+
+        sender, receiver, amount = transfer_events[0].arguments
+        self.assertEqual(owner_script_hash, sender)
+        self.assertEqual(test_account_3_script_hash, receiver)
         self.assertEqual(10_000_000, amount)
```

Comment:
since we're refactoring this tests, could you change this number constant to a variable?
```suggestion
        transfer_amount_3 = 10_000_000  # is this supposed to be 1 NEO? NEO doesn't have decimals anymore
        invokes.append(runner.call_contract(path, 'transfer', owner_script_hash, test_account_3_script_hash,
                                            transfer_amount_3, None))
        expected_results.append(True)
        runner.execute(account=self.OWNER)
        self.assertEqual(VMState.HALT, runner.vm_state, msg=runner.error)

        runner.run_contract(path, 'transfer', owner_script_hash, test_account_3_script_hash, transfer_amount_3,
                            None, account=self.OWNER)

        transfer_events = runner.get_events('Transfer')
        self.assertEqual(1, len(transfer_events))
        self.assertEqual(3, len(transfer_events[0].arguments))

        sender, receiver, amount = transfer_events[0].arguments
        self.assertEqual(owner_script_hash, sender)
        self.assertEqual(test_account_3_script_hash, receiver)
        self.assertEqual(transfer_amount_3, amount)
```